Assets/i5 Toolkit for Unity/Runtime/OpenID Connect/Scripts/OpenIDConnectService.cs

```diff
@@ -147,6 +147,8 @@ public void OpenLoginPage()
                 return;
             }
 
+            ServerListener.ListeningUri = "http://127.0.0.1:52229/";
+

```

Comment:
I think it would be best to move this line into the samples as it would now affect every single implementation, no matter whether they use the Google login or another one.

The samples folder can be found by renaming the Samples~ folder to Samples. Unity ignores all folders which end in a tilde. To rename the folder, please use the command "git mv Samples~ Samples" because just renaming it in the file system will purge the history of these files. After renaming it, the Samples folder will appear in Unity. In the login sample, you can create a sample similar to the login samples for Learning Layers and GitHub. In this sample, you can specify in the setup script, that the listening uri needs to be fixed to a specific port.

Reply:
We have created a sample for the Google Login as suggested and removed the highlighted line from the OpenIDConnectService.